playground/mocks/spec-duo/duo-iframe.html

```diff
@@ -0,0 +1,6 @@
+<html>
```

Comment:
style-nit: Instead of the `spec-*` prefix, can we utilize the path: `playgrounds/mock/duo/iframe.html` for this file?

Reply:
Haisheng suggested in a previous comment to name the folder `spec-duo`. I get your point, this is not really a spec file, but I think it makes sense to keep both files use to mock duo in the same directory.